collector/printer_object.go

```diff
@@ -44,26 +44,26 @@ type PrinterObjectStatus struct {
 
 type PrinterObjectMcu struct {
 	LastStats struct {
-		McuAwake float64 `json:"mcu_awake"`
-		// McuTaskAvg float64 `json:"mcu_task_avg"` // value returned in the format 1.5e-05
-		// McuTaskStddev float64 `json:"mcu_task_stddev"` // value returned in the formate 1e-05
-		BytesWrite      float64 `json:"bytes_write"`
-		BytesRead       float64 `json:"bytes_read"`
-		BytesRetransmit float64 `json:"bytes_retransmit"`
-		BytesInvalid    float64 `json:"bytes_invalid"`
-		SendSeq         float64 `json:"send_seq"`
-		ReceiveSeq      float64 `json:"receive_seq"`
-		RetransmitSeq   float64 `json:"retransmit_seq"`
-		Srtt            float64 `json:"srtt"`
-		Rttvar          float64 `json:"rttvar"`
-		Rto             float64 `json:"rto"`
-		ReadyBytes      float64 `json:"ready_bytes"`
-		StalledBytes    float64 `json:"stalled_bytes"`
-		Freq            float64 `json:"freq"`
-	} `json:"last_stats"`
+		McuAwake        float64 `mapstructure:"mcu_awake"`
+		McuTaskAvg      float64 `mapstructure:"mcu_task_avg"`
+		McuTaskStddev   float64 `mapstructure:"mcu_task_stddev"`
+		BytesWrite      float64 `mapstructure:"bytes_write"`
+		BytesRead       float64 `mapstructure:"bytes_read"`
+		BytesRetransmit float64 `mapstructure:"bytes_retransmit"`
+		BytesInvalid    float64 `mapstructure:"bytes_invalid"`
+		SendSeq         float64 `mapstructure:"send_seq"`
+		ReceiveSeq      float64 `mapstructure:"receive_seq"`
+		RetransmitSeq   float64 `mapstructure:"retransmit_seq"`
+		Srtt            float64 `mapstructure:"srtt"`
+		Rttvar          float64 `mapstructure:"rttvar"`
+		Rto             float64 `mapstructure:"rto"`
+		ReadyBytes      float64 `mapstructure:"ready_bytes"`
+		StalledBytes    float64 `mapstructure:"stalled_bytes"`
+		Freq            float64 `mapstructure:"freq"`
+	} `mapstructure:"last_stats"`
 }
 
-const mcuQuery string = "mcu=last_stats"
+//const mcuQuery string = "mcu=last_stats"
```

Comment:
this line can be removed instead of commented out

Reply:
Ah, missed this!